packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -56,11 +66,26 @@
                     : undefined,
         },
     ]
-
-    let modal: Modal
-    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
     $: $selectedAccountVestingPayouts, (timeUntilNextPayout = getTimeUntilNextPayout())
 
+    onMount(() => {
+        getMinRequiredStorageDeposit()
+        areOutputsReadyForConsolidation()
+    })
+
+    function areOutputsReadyForConsolidation(): void {
+        $selectedAccount
+            .prepareConsolidateOutputs({ force: false, outputThreshold: 2 })
+            .then(() => (hasOutputsToConsolidate = true))
+            .catch(() => (hasOutputsToConsolidate = false))
+    }
+
+    function getMinRequiredStorageDeposit() {
+        getRequiredStorageDepositForMinimalBasicOutput()
+            .then((deposit) => (minRequiredStorageDeposit = deposit))
+            .catch(() => (minRequiredStorageDeposit = 0))
```

Comment:
catching an error and using 0 as the value is going to give problems because you have `$selectedAccount?.balances?.baseCoin?.available < minRequiredStorageDeposit` to disable the button, if this is 0 and I have 1micro, the condition would not be right, I feel like we shouldnt no allow collecting if you cant get the min storage deposit